config/crd/bases/search.open-cluster-management.io_searchoperators.yaml

```diff
@@ -1,8 +1,6 @@
-# Copyright (c) 2021 Red Hat, Inc.
-# Copyright Contributors to the Open Cluster Management project
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why is this version changing?

Reply:
Still working on this .. incorrect versions getting pulled @jlpadilla